src/controllers/pet.controller.ts

```diff
@@ -446,6 +448,7 @@ export const queryList = async (req: any, res: any) => {
         country: 1,
         history: 1,
         category: 1,
+        activityLevel: 1,
```

Comment:
Hi, sorry for this change. 😀
Please, delete activityLevel because I added it recently

Thank you 😀

Reply:
I saw that you'd added this almost at the same time I opened my PR 😄 

But I belive that you can safely merge this since we've added the excact code at the same line, and then git will figure out that it should only be one. Otherwise i will fix it when i get home from work 👍

Reply:
I've merged upstream/master into my branch. So Its should be fixed now.

Reply:
Nice! I'm going to merge.
If there is a problem or duplicate, I will fix it, don't worry. 

😄👍